drivers/video/isx012.c

```diff
@@ -694,6 +713,8 @@ static int isx012_putreg(isx012_dev_t *priv,
 
   memcpy((uint8_t *)&buffer[2], (uint8_t *)&regval, regsize);
 
+  i2c_lock();
```

Comment:
should i2c_ops_s::transfer guarantee the safe concurrent operation, not the caller.